pkg/util/httpproxy/upgradeawareproxy.go

```diff
@@ -35,11 +36,13 @@ func NewUpgradeAwareSingleHostReverseProxy(clientConfig *kclient.Config, backend
 	if err != nil {
 		return nil, err
 	}
+	reverseProxy := httputil.NewSingleHostReverseProxy(backendAddr)
+	reverseProxy.FlushInterval = 50 * time.Millisecond
```

Comment:
given that there are likely to be a lot of long-lived requests going through this proxy, I'm wondering if we should raise this to 100-200 ms